schema/annotation-source.yaml

```diff
@@ -67,12 +68,15 @@ $defs:
         const: Contribution
         default: Contribution
         description: MUST be "Contribution".
-      agent:
+      contribution_made_by:
         $ref: "#/$defs/Agent"
-      date:
+      started:
```

Comment:
VA model provides separate started and ended properties.  If an implementation wants to just provide a single data/time - the VA spec recommends using 'ended'.

However, we have gone back and forth on this design pattern/convention, and are open to changing it (e.g. if people think a single date is always sufficient, we can just provide a 'date' or 'date_performed' attribute.

Reply:
I've seen this issue many times in the past for data models involving dates/times.  They refer to the entities that have a notion of time to be Moment/Intervals. This means they are either a moment in time (one date/time field) or an Interval (start/end dates/times).  I think we should determine if the entity is a Moment or an Interval type overall and pick the appropriate date or range approach.  This seems to follow a similar pattern to our Number and Range for precise and range-like positions on the sequence.

Reply:
Agreed - lets discuss and choose an approach. I suspect that four our purposes, a 'Moment' as you call it (i.e. a single date or datetime) is sufficient - and that we are not concerned with knowing that, for example, some contribution began on one day and ended on another. But if we are, or might be at some point, we should define a model that accommodates such precision/expressiveness.